src/psyclone/tests/domain/gocean/transformations/gocean1p0_transformations_test.py

```diff
@@ -225,23 +222,27 @@ def test_omp_parallel_loop(tmpdir):
     gen = gen.lower()
     expected = ("!$omp parallel do default(shared), private(i,j), "
                 "schedule(static)\n"
-                "      do j=2,jstop\n"
-                "        do i=2,istop+1\n"
+                "      do j = 2, jstop, 1\n"
+                "        do i = 2, istop+1, 1\n"
                 "          call compute_cu_code(i, j, cu_fld%data, "
                 "p_fld%data, u_fld%data)\n"
                 "        end do\n"
                 "      end do\n"
                 "      !$omp end parallel do")
     assert expected in gen
 
+    psy, invoke = get_invoke("single_invoke_three_kernels.f90", API, idx=0,
```

Comment:
Would be nice if we could avoid parsing a file from scratch again here.

Reply:
I found some errors reusing it, so for the time being I added relevant TODOs that may fix them.

Reply:
Removed the TODO, using fortran_writer this is fixed and we don't need to repeat the parsing.